packages/react/src/portal.tsx

```diff
@@ -68,7 +74,7 @@ export const Portal: (props: PortalProperties & RefAttributes<ComponentInternals
               <group onPointerOver={() => null} />
             </ChildrenToFBO>,
             vScene,
-            injectState,
+            { ...injectState, camera: camera },
```

Comment:
You're recreating the state on every react re-render

Reply:
Are you suggesting we should replace the line with something like this:
`useMemo(() => ({ ...injectState, camera: camera }), [injectState, camera]),`
Or do you envision something better ? :)

Reply:
you can just create the camera inside the useMemo for creating the "injectState" which is semantically more cleaner (creating the injectState in one place) and more performant (1 instead of 3 useMemo)

Reply:
Clean! I like it.